Library/Homebrew/cask/artifact/keyboard_layout.rb

```diff
@@ -0,0 +1,36 @@
+# typed: true
+# frozen_string_literal: true
+
+require "cask/artifact/moved"
+
+module Cask
+  module Artifact
+    # Artifact corresponding to the `keyboard_layout` stanza.
+    #
+    # @api private
+    class KeyboardLayout < Moved
+      extend T::Sig
+
+      sig { returns(String) }
+      def self.english_name
+        "Keyboard Layout"
+      end
+
+      def install_phase(**options)
+        super(**options)
+        delete_keyboard_layout_cache(**options)
+      end
+
+      def uninstall_phase(**options)
+        super(**options)
+        delete_keyboard_layout_cache(**options)
+      end
+
+      private
+
+      def delete_keyboard_layout_cache(command: nil, **_)
+        command.run!("/bin/rm", args: ["-f", "--", "/System/Library/Caches/com.apple.IntlDataCache.le*"], sudo: true)
+      end
```

Comment:
Does this need to take a command argument if it's always going to run the same command?

Reply:
The `command` argument here contains more information than calling `system_command!` directly, e.g. whether it should be verbose, so that all commands for a given cask behave the same way in terms of output.

Reply:
Ah, I looked at the other artifacts and see that this is a common pattern. I wonder why it accepts nil as an argument though since it seems to be require here and I couldn't find any references where it would be nil.